### ISSUE TITLE
finetune on hmdb51, low accuracy on val set

### ISSUE THREAD

**Issue by gwh0112** [kenshohara/3D-ResNets-PyTorch#54]
Hi, I finetune the pretrained resnet-34-kinetics model on hmdb51, with the following command:
python main.py --root_path ~/data --video_path hmdb51/jpg --annotation_path hmdb51_1.json --result_path resnet34_finetune_hmdb51_results --dataset hmdb51 --n_classes 400 --n_finetune_classes 51 --pretrain_path models/resnet-34-kinetics.pth --ft_begin_index 4 --model resnet --model_depth 34 --resnet_shortcut A --batch_size 128 --n_threads 4 --checkpoint 5
When I check the performance on train and val sets, accuracy(train) is kind of high - around 0.8, while accuracy(val) is always around 0.5. Is this correct? It seems the training overfits into the train set?

**Comment by kenshohara** [comment]
Because HMDB-51 is a relatively small dataset, 
the training on it tends to overfit.
My validation (clip) accuracy of resnet-50 is about 0.57 at epoch 300.

**Comment by FelixZhang7** [comment]
@gwh0112 excuse me,i tried this code,it runs fast but i got 0 loss and 0 accuracy in hmdb51,the  train.log and val.log seems wrong,i donnot know why

**Comment by gwh0112** [comment]
@FelixZhang7 I'm not sure about your issue. But please double-check whether the data are loaded correctly or not.

**Comment by Masterzhuior** [comment]
> @gwh0112 excuse me,i tried this code,it runs fast but i got 0 loss and 0 accuracy in hmdb51,the train.log and val.log seems wrong,i donnot know why

I had the same problem as you. Did you fix this problem??
